### PR TITLE
Pointing chart to gateway 3.3.1

### PR DESCRIPTION
**Pull request by BStarmerSmith**
Pointing chart to gateway 3.3.1.

### CODE REVIEW

**Comment by sderosiaux** [comment]
i don't think charts/gateway/conduktor-gateway-3.3.0.tgz should be pushed?

**Comment by BStarmerSmith** [comment]
> i don't think charts/gateway/conduktor-gateway-3.3.0.tgz should be pushed?

Yep mistake from where I was doing some local CS testing! didn't spot it, thank you!

**Comment by sderosiaux** [comment]
> > i don't think charts/gateway/conduktor-gateway-3.3.0.tgz should be pushed?
> 
> Yep mistake from where I was doing some local CS testing! didn't spot it, thank you!

no worries. don't forget you can use `.gitignore` ;-)

**Review by sderosiaux — APPROVED** [review]

**Review by trobert — APPROVED** [review]
:pray:

File: charts/gateway/index.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    - name: Bitnami
+      url: https://github.com/bitnami/charts
+    name: kafka
+    sources:
+    - https://github.com/bitnami/containers/tree/main/bitnami/kafka
+    - https://kafka.apache.org/
+    urls:
+    - charts/kafka-18.5.0.tgz
+    version: 18.5.0
+generated: "2024-09-26T15:43:44.189741+01:00"
```
Comment (qboileau) [inline charts/gateway/index.yaml:89]: This should be removed

File: charts/gateway/Chart.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: "3.3.0"
+appVersion: "3.3.1"
 description: Conduktor Gateway chart
 name: conduktor-gateway
-version: 3.3.0
+version: 3.3.1
```
Comment (qboileau) [inline charts/gateway/Chart.yaml:5]: Do not increment the chrat version here, it will be done on create release workflow

**Review by qboileau — APPROVED** [review]

**Pull request merged**